week2/assignment/가계부 💸/js/History/TransactionFilter.js

```diff
@@ -0,0 +1,28 @@
+import { TRANSACTION_TYPE, HISTORY_LIST } from "../utils/constants.js";
+import { INCOME_BUTTON, SPENDING_BUTTON } from "../utils/documentElements.js";
+import { renderHistoryList, resetData } from "./HistoryView.js";
+
+INCOME_BUTTON.addEventListener("click", handleButtonClick);
+SPENDING_BUTTON.addEventListener("click", handleButtonClick);
+
+function handleButtonClick() {
+  const isIncomeChecked = INCOME_BUTTON.checked;
+  const isSpendingChecked = SPENDING_BUTTON.checked;
+
+  if (!isIncomeChecked && !isSpendingChecked) {
+    // 둘 다 체크 안된 경우
+    resetData();
+  } else {
+    const selectedTypes = [];
+
+    if (isIncomeChecked) selectedTypes.push(TRANSACTION_TYPE.INCOME);
```

Comment:
요런거는 이항 연산자로 바꿀 수 있을 것 같ㅇㅏ!
```suggestion
    isIncomeChecked && selectedTypes.push(TRANSACTION_TYPE.INCOME);
```